web/pageObjects/login_page.ts

```diff
@@ -0,0 +1,78 @@
+import * as defaultPage from "../pageObjects/default_page"
+
+const btnSignInWithEmail = '//main//button[text()="Sign in with email"]'
+const signInHeader = '//main//h1'
+const btnSignIn = '//main//button[@type="submit"]'
+const enterEmailField = '//main//input[@name="email"]'
+const enterPassField = '//main//input[@name="password"]'
+
+const incorrectEmailOrPassMsg = '//main//div//form//h3'
```

Comment:
gali sutrumpint selectoriu be div'o //main//form//h3

Reply:
Sutrumpinta